arduino-ide-extension/src/node/core-service-impl.ts

```diff
@@ -227,11 +229,12 @@ export class CoreServiceImpl extends CoreClientAware implements CoreService {
         result.on('end', () => resolve());
       });
     } catch (e) {
+      const errorMessage = `Error while burning the bootloader: ${e.details}`;
```

Comment:
I know errors are not localized (yet)... but they should.. 

Maybe we can start adopting i18n for new errors

Reply:
@silvanocerza btw the PR is good. 🪄